flink-ml-lib/src/main/java/org/apache/flink/ml/feature/splitter/Splitter.java

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
In order to align the functionality of this algorithm with Spark's `DataSet.randomSplit`, do you think the following API would be better?
- rename the algorithm from `Splitter` to `RandomSplit`.
- The parameter of this algorithm is a `double[]` whose length >= 2 and contains positive values. The positive values are the weights, proportional to the number of records to be placed in the split datastreams.
  - If the sum of the values in the double array is not 1, they will be normalized.
- The output is a Table array whose size is equal to the double array parameter's length.